ruby_on_rails/sidekiq.md

```diff
@@ -0,0 +1,82 @@
+# Sidekiq
+
+* Add the following gem `gem 'sidekiq'` and install it
```

Comment:
I would explain quickly what is this section about (background jobs) and the fact that we use sidekiq instead of suckerpunch because suckerpunch causes memory issues on Heroku and add a link to the github page of the gem